<?xml version="1.0"?>
<launch>

	<param name="/use_sim_time" value="true" />


  <node name="cloud_controller" pkg="gpar_lidar" type="cloud_controller" output="screen">
    <param name="input_cloud" type="str" value="cloud_ldmrs" />
   </node>


  <!-- This bag has to have immediate lidar scan and correspondent transforms! -->
    <node name="playbag" pkg="rosbag" type="play"
      args="$(arg bag_filename) --pause --clock" output="screen" />

      <node name="rviz" pkg="rviz" type="rviz" required="true"
      args="-d $(find gpar_lidar)/rviz/ldmrs.rviz" />

   <node name="resetpoints" pkg="rosservice" type="rosservice" args="call --wait /cloud_controller/command_parser 2" />
   <node name="savepoints" pkg="rosservice" type="rosservice" args="call --wait /cloud_controller/command_parser 1" />

</launch>

